src/storage/mutate/UpdateEdgeProcessor.cpp

```diff
@@ -171,17 +172,46 @@ kvstore::ResultCode UpdateEdgeProcessor::collectEdgesProps(
         if (constSchema == nullptr) {
             return kvstore::ResultCode::ERR_UNKNOWN;
         }
+        auto schema = std::const_pointer_cast<meta::SchemaProviderIf>(constSchema);
```

Comment:
You could tweak the declaration of `RowUpdater`, so that this `const_cast` is no longer needed. And `constSchema` can be renamed to `schema`.